### ISSUE TITLE
Usage: Internal DB reference in description of VM snapshot on primary

### ISSUE THREAD

**Issue by weizhouapache** [apache/cloudstack#5666]
see cmk results below

```
(localcloud) SBCM5> > list usagerecords type=27 startdate=2021-11-03 enddate=2021-11-03
{
  "count": 82,
  "usagerecord": [
    {
      "account": "admin",
      "accountid": "0e3e52ed-03db-11ec-8879-1e0079000360",
      "description": "VMSnapshot on primary storage usage Id: 1 (2790057b-5a2f-4f90-ae5b-021573d0943d)  for VM L2 (7a25841b-cfac-49ca-90eb-0cb187ba3414) with size (8.00 GB) 8589934592",
      "domain": "ROOT",
      "domainid": "e5ce2438-03da-11ec-8879-1e0079000360",
      "enddate": "2021-11-03'T'17:11:53+00:00",
      "name": "i-2-1000-VM_VS_20211103170813",
      "rawusage": "0.058435",
      "size": 8589934592,
      "startdate": "2021-11-03'T'17:08:23+00:00",
      "tags": [],
      "usage": "0.058435 Hrs",
      "usageid": "2790057b-5a2f-4f90-ae5b-021573d0943d",
      "usagetype": 27,
      "virtualmachineid": "7a25841b-cfac-49ca-90eb-0cb187ba3414",
      "zoneid": "59260b3d-f587-4a8c-8c58-182ae3f01679"
    },

...
```

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
Usage
~~~

##### CLOUDSTACK VERSION

~~~
4.16.0.0
~~~

##### STEPS TO REPRODUCE

~~~
1. change global settings for usage server , restart cloudstack management and usage server

enable.usage.server = true
usage.stats.job.aggregation.range = 5

2. create vm;
3. create vm snapshot
4. wait for 5-10 minutes, check usagerecord by cmk or cloud_usage.cloud_usage table (usage type=25).
~~~

##### EXPECTED RESULTS

~~~
the `description` in cmk result does not contain the usage id.


usage Id (2790057b-5a2f-4f90-ae5b-021573d0943d)

~~~

##### ACTUAL RESULTS

~~~
```
usage Id: 1 (2790057b-5a2f-4f90-ae5b-021573d0943d)
```
~~~

**Comment by weizhouapache** [comment]
If possible, please also check the description of all other usage types !

**Comment by DaanHoogland** [comment]
> If possible, please also check the description of all other usage types !

I checked the code, the only other place where Id is used Uuid is mentioned as well. I supose this was on purpose and anm leaving it as is.

**Comment by DaanHoogland** [comment]
merged in #5738
